pkg/pool-gyro/contracts/Gyro2CLPPool.sol

```diff
@@ -115,8 +126,7 @@ contract Gyro2CLPPool is IGyro2CLPPool, BalancerPoolToken {
         (uint256 virtualParamIn, uint256 virtualParamOut) = _getVirtualOffsets(
             balanceTokenInScaled18,
             balanceTokenOutScaled18,
-            tokenInIsToken0,
-            request.kind
+            tokenInIsToken0
```

Comment:
What is this argument change in `_getVirtualOffsets`? Is something not updated, or did another change slip in?

Reply:
I think `kind` is unused now that we are not rounding virtual offsets differently.